dev-python/pypeg2/pypeg2-2.15.2.ebuild

```diff
@@ -20,6 +20,7 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
 RDEPEND="dev-python/lxml[${PYTHON_USEDEP}]"
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
There is no BDEPEND with EAPI-5. It's 7+ thing. So only modify PYTHON_COMPAT in this ebuild and don't add / touch anything else.

Reply:
Ok